plugin/output/s3/s3.go

```diff
@@ -74,13 +82,28 @@ type Config struct {
 	Endpoint  string `json:"endpoint" required:"true"`
 	AccessKey string `json:"access_key" required:"true"`
 	SecretKey string `json:"secret_key" required:"true"`
-	Bucket    string `json:"bucket" required:"true"`
-	Secure    bool   `json:"secure" default:"false"`
+	// Required s3 default bucket.
+	Bucket string `json:"bucket" required:"true"`
+	// MultiBuckets is additional buckets, which can also receive event.
+	// Event must contain `bucket_name` which value is valid s3 bucket name.
+	// Events without `bucket_name` sends to	 Bucket.
+	MultiBuckets []singleBucket `json:"multi_buckets" required:"false"`
```

Comment:
We should support any bucket name that comes from field value. Hardcode here is very inconvenient for the user

Reply:
to be done

Reply:
A added both